### DISCUSSION TITLE
Alpine Expression Error: Cannot read properties of undefined (reading 'entangle')

### DISCUSSION THREAD

**Discussion by soyCristianRico** [livewire/livewire#6672] [1. Help] [answered]
I'm having issues with the communication between Alpine and Livewire. It tells me that it can't read the @entangle. But what's strange is that the popup shows up and disappears when I modify its value in the Livewire component, so it's kind of weird. However, I don't want to leave these error messages in the console.

![image](https://github.com/livewire/livewire/assets/96917793/ae5d012d-1688-403f-aaa1-8216ca37fdb2)

I also can't get the $this->reset('email'); to work in the Livewire component. Maybe it's because of this, I don't know...

Layout:

![image](https://github.com/livewire/livewire/assets/96917793/a354bed3-d2dc-4073-8a4c-4ea5475bf8d2)

Livewire view:

![image](https://github.com/livewire/livewire/assets/96917793/52be414c-e05f-48d2-aea0-d129fa4c388a)

Livewire component:

![image](https://github.com/livewire/livewire/assets/96917793/ab793595-e501-44a9-9caf-54d1144f8e6c)

I installed it with Laravel Breeze. I've already been looking for solutions online and haven't found an answer.

If instead of @entangle I manually set a value, no error appears. But the $this->reset('email'); still doesn't work.

I tried installing it with the Tall Stack Preset, but the error kept coming up.

Thanks for your help.

**Comment by joshhanley** [comment]
@soyhacedor are you using V2 or V3? Also don't paste screenshots of code, as we can't run them, instead paste code in code blocks.

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Thanks!

**Comment by Kovah** [comment]
Hey @soyhacedor, I hade the same issue and solved it by double-checking that Livewire was correctly loaded _together_ with Alpine. On the page where the error occurred, Alpine was loaded in the `<head>` of the page and not by using the `@livewireScripts` helper. For whatever reason, Livewire components are not properly recognized if you load Alpine and Livewire separately.

**Comment by soyCristianRico** [comment, marked as answer]
Sorry @joshhanley, first time posting on Github and I don't know how it's going.

I have managed to solve it using the Livewire layout: "php artisan livewire:layout"

It seems that it was not a version problem, but rather how the resources were being loaded.

Thank you!

**Reply by aneeshsudhakaran-git** [reply]
hello, I'm facing same issue. what change you did in layouts? thanks in advance.

**Comment by ewatch** [comment]
Another source of the problem which was the problem for my project (using livewire since earlier versions)

Please check the files:
- bootstrap.js (it should not contain "require" statements anymore, check against this file here: https://github.com/laravel/laravel/blob/10.x/resources/js/bootstrap.js)
- app.js (it may still contain Alpine.js imports which is not necessary anymore, as it will be injected by livewire)

Hope it helps!
